packages/testing/src/web/fileMock.ts

```diff
@@ -4,3 +4,4 @@
  * https://jestjs.io/docs/en/webpack#handling-static-assets
  */
 export default 'fileMock'
+// TODO(jgmw): Ask for clarification on the above
```

Comment:
We still want this because it's used in the web side's jest configuration. We can just update the comments above to explain that it's just mocking a bundler's import of certain static assets (like jpg, png, etc.)

See https://github.com/redwoodjs/redwood/blob/main/packages/testing/config/jest/web/jest-preset.js#L77-L78